src/installer/tests/Assets/TestProjects/LightupLib/LightupLib.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <!-- Using the latest TargetFramework that is supported by the SDK until we can build against the live targeting pack. -->
+    <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
```

Comment:
Doesn't this mean that installer tests will no longer run against live libraries bits? I don't know if they currently do. @jkoritzinsky ?

Reply:
These test projects run against the live runtime. We should use NETCoreAppCurrent and restore them against the ref pack that we build in the packs.product subset.

Reply:
cc @ViktorHofer

Reply:
> These test projects run against the live runtime. We should use NETCoreAppCurrent and restore them against the ref pack that we build in the packs.product subset.

Master currently uses the 5.0-RC1 targeting and runtime pack: https://github.com/dotnet/runtime/blob/543cb488437ea0b05a58e3297214a0c9fb2064ea/src/installer/tests/Assets/TestUtils/TestProjects.targets#L20. If I got this right, then https://github.com/dotnet/runtime/blob/543cb488437ea0b05a58e3297214a0c9fb2064ea/src/installer/tests/Directory.Build.targets#L16 is publishing the entire bundle to disk which is then used for the test invocation.

Based on that I don't see how changing the tfm would impact whether we run against the live built runtime. I'm not an expert on the installer subset so please correct me if I'm wrong here.

Reply:
I'd prefer if we can get the targeting pack version that the projects build against to match the live built runtime so we're correctly testing what we mean to test without using rollforward.

Reply:
Absolutely, we should follow-up on this. My comment was about master already using a different TargetingPackVersion that what is used during runtime which suggests that this PR shouldn't address this.